docs/completions.rst

```diff
@@ -0,0 +1,63 @@
+.. highlight:: console
+
+===============
+Bash Completion
```

Comment:
I'd change here to `Shell Completion`, since we are talking about different kinds of shells.

Reply:
Agree